docs/channels/qiscus.md

```diff
@@ -0,0 +1,59 @@
+---
+id: channel-qiscus
+title: Qiscus
+---
+
+Qiscus is an AI compliant, secure and flexible chat SDK for apps and websites. They also provide multichannel customer service chat solution which is now getting popular as it is best designed for everyone to enhance their customer service experiences using multiple channels and bot-human collaboration customer service. In this section, we will learn on how to connect our bot with Qiscus thorough these following steps.
+
+## Create Qiscus Channel
+
+1. First thing first, you need to register yourself at [Qiscus Multichannel CS Chat Website](https://multichannel.qiscus.com) to create a new app. You will be asked to fill in several information regarding your account.
+![Register](./images/qiscus/sensor_register.jpg)
+
+2. As soon as you signed up, you will get an email verification from Qiscus Multichannel CS Chat that you need to verify before you can use it. So just check your email and verify it.
+![Verify](./images/qiscus/validate.png)
+
+3. After you verified your account, you can log into the Qiscus Multichannel CS Chat dashboard via [login page](https://multichannel.qiscus.com). Please fill in your credentials to get into the dashboard.
+
+4. Once you logged in, go to the integration page, and enable the "Qiscus Widget" tab like shown on picture below.
+![Qiscus Widget Integration](./images/qiscus/red_qiscuswelcomepage.png)
+
+5. Copy the snippet code shown on the page and paste it into your website html. It is not set yet, you can open your html later after you integrate the bot.
```

Comment:
```suggestion
5. Copy the snippet code shown on the page and paste it into your _**website html**_. It is not set yet, you can open your html later after you integrate the bot.
```

`It is not set yet,` itu mksdnya gimana?

Reply:
maksudnya belom selesai gitu prosesnya, bagusnya kalimatnya gimana yak

Reply:
`you will use your html code later blabla`